Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG image_tag=latest
```

Comment:
a branching model could influence this, but as long as there is only `master` it makes sense for the default tag to be `latest` (which isn't necessarily pushed when built)